source/_includes/common-tasks/commandline.md

```diff
@@ -58,7 +58,7 @@ Replace x.y.z with the desired version like `--version {{current_version}}`
 
 You can get a better description of the CLI capabilities by typing `ha help`:
 
-```bash
+```
```

Comment:
There must be a language flag set.

Reply:
Why is that? For example in the github docs (https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) I read "You can add an *optional* language identifier. I coudn't find a markdown guide related to code formatting for home assistant. If there is one, might you point me to it? https://developers.home-assistant.io/docs/documenting/standards doesn't seem to cover this.

Reply:
This is not GitHub pages or docs. So their documentation does not apply here.
Or CI guards it, to prevent it is begin forgotten (which happened a lot).

In this case, just setting it to `txt` might be fitting.

Reply:
Of course I understand that this is not GitHub pages, I referenced it only as a (widely used) example. But I understand why you want to leave this check in the CI. Changing it to "txt" seems to work fine.